### PR TITLE
migrate tslint to eslint

### PR DESCRIPTION
**Pull request by VOSpdUkraine**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -143,9 +143,13 @@
           },
           "configurations": {
             "production": {
-              "browserTarget": "ew-boilerplate-ui:build:production"
```
Comment (dawidgil) [inline angular.json:146]: Well, running `ng serve --prod` causes here an error. You shouldn't change project name here. If you want change a project name, then you need to change it everywhere in this file.

File: angular.json [hunk 2]
```diff
@@ -184,15 +188,11 @@
...
-            "tsConfig": [
-              "tsconfig.app.json",
-              "tsconfig.spec.json",
-              "e2e/tsconfig.json"
-            ],
-            "exclude": [
-              "**/node_modules/**"
+            "lintFilePatterns": [
+              "src/**/*.ts",
+              "src/**/*.html"
```
Comment (dawidgil) [inline angular.json:195]: excluding node_modules is not needed here?

File: package.json [hunk 3]
```diff
@@ -109,6 +117,9 @@
     "tslint": "~6.1.0",
```
Comment (dawidgil) [inline package.json:117]: Please remove tslint as it is not needed anymore.

File: package.json [hunk 4]
```diff
@@ -90,6 +95,9 @@
     "@types/jasminewd2": "2.0.3",
     "@types/lodash": "^4.14.118",
     "@types/node": "^12.11.1",
+    "@typescript-eslint/eslint-plugin": "4.28.2",
+    "@typescript-eslint/parser": "4.28.2",
+    "eslint": "^7.26.0",
     "codelyzer": "^5.1.2",
```
Comment (dawidgil) [inline package.json:101]: I think codelyzer is not needed anymore. As far as i know it was used by tslint.

File: src/app/routes/applications/governance-list/governance-list.component.html [hunk 5]
```diff
@@ -1,7 +1,6 @@
 <!-- Filter Sub Organization -->
-
 <div class="d-flex align-items-center flex-wrap mb-2" *ngIf="listType === ListType.ORG && orgHierarchy?.length">
-  <div class="d-flex align-items-center my-1" *ngIf="i !== (orgHierarchy?.length - 1)">
```
Comment (dawidgil) [inline src/app/routes/applications/governance-list/governance-list.component.html:4]: Are you sure this won't break functionality?

**Review by dawidgil — COMMENTED** [review]
Few things that i think are needed to adjust.
Please update also package-lock.json file.

File: package.json (hunk 3 above)
Comment (VOSpdUkraine) [reply package.json:117]: updated

File: angular.json (hunk 2 above)
Comment (VOSpdUkraine) [reply angular.json:195]: updated

File: angular.json (hunk 1 above)
Comment (VOSpdUkraine) [reply angular.json:146]: yes it was my mistake fixed

File: src/app/routes/applications/governance-list/governance-list.component.html (hunk 5 above)
Comment (VOSpdUkraine) [reply src/app/routes/applications/governance-list/governance-list.component.html:4]: updated

**Review by dawidgil — APPROVED** [review]

**Pull request merged**